boards/arm/rpi_pico/rpi_pico-common.dtsi

```diff
@@ -145,3 +145,4 @@ zephyr_udc0: &usbd {
 };
 
 pico_spi: &spi0 {};
+pico_i2c: &i2c1 {};
```

Comment:
The i2c1 is not defaultly available.
It should move to .overlay file.

Reply:
Thanks, will make these changes this evening :)